src/cms/api/park-sub-page/models/park-sub-page.js

```diff
@@ -5,4 +5,20 @@
  * to customize this model
  */
 
-module.exports = {};
+const regex = new RegExp("^[a-z0-9/]+(?:-[a-z0-9/]+)*$")
```

Comment:
Is this one intentionally different from the other?

Reply:
Yes, it's because the slug field in `parkSubPage` is required. So it shouldn't allow an empty string.